ld: warning: directory '/usr/local/opt/openssl/lib' following -L not found

--- src/Makefile	2024-04-16 12:18:47.000000000 +0800
+++ src/Makefile	2024-05-04 12:03:20.000000000 +0800
@@ -158,13 +158,6 @@
 	# Homebrew's OpenSSL is not linked to /usr/local to avoid
 	# conflicts with the system's LibreSSL installation so it
 	# must be referenced explicitly during build.
-ifeq ($(uname_M),arm64)
-	# Homebrew arm64 uses /opt/homebrew as HOMEBREW_PREFIX
-	OPENSSL_PREFIX?=/opt/homebrew/opt/openssl
-else
-	# Homebrew x86/ppc uses /usr/local as HOMEBREW_PREFIX
-	OPENSSL_PREFIX?=/usr/local/opt/openssl
-endif
 else
 ifeq ($(uname_S),AIX)
         # AIX
